.github/workflows/master.yml

```diff
@@ -0,0 +1,33 @@
+name: "Staging / master branch"
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  run-all-tests:
+    uses: ./.github/workflows/_tests.yml
+
+  deploy-locksmith-staging:
```

Comment:
We should also deploy all netlify services, correct?

Reply:
yep, fixing the workflow for PRs first, then will add